projects/golang/go/docker/debianBase/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM buildpack-deps:bullseye-scm
+
+ARG GOLANG_ARCHIVE_PATH
+
+ADD $GOLANG_ARCHIVE_PATH /usr/local
+
+ENV GOPATH /go
+ENV PATH /usr/local/go/bin:$PATH
+ENV PATH $GOPATH/bin:$PATH
+ENV GOLANG_VERSION GOLANG_VERSION
+
+RUN set -eux; \
```

Comment:
we're running this all as a single `RUN` command to avoid space-issues in pre-submit; if we have too many layers we'll run into disk space issues on the fargate builder nodes